apph-front/src/static/components/UserTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { IUserTable } from '../../utils';
+import * as React from 'react';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { useTranslation } from 'react-i18next';
+
+interface userTableProps {
+  data: IUserTable[];
+  loading: boolean;
+}
+
+export const UserTable = ({ data, loading = false }: userTableProps) => {
+  const { t } = useTranslation();
```

Comment:
`jest.mock('react-i18next', () => ({
  // this mock makes sure any components using the translate hook can use it without a warning being shown
  useTranslation: () => {
    return {
      t: (str: string) => str,

      i18n: {
        changeLanguage: jest.fn()
      }
    };
  }
}));`

Reply:
j'ai pas compris où est le problème ?